CONTRIBUTING.md

```diff
@@ -51,6 +51,8 @@ There are also a couple of nice to haves:
 * **Environment:** You can find this with `jina --version-full`
 * **Screenshots:** If they're relevant
 
+To understand how our issues are labeled, check out our [issue label guide](https://github.com/jina-ai/jina/blob/master/github_issue_label_guide.md).
```

Comment:
```suggestion
To understand how our issues are labeled, check out our [issue label guide](./github_issue_label_guide.md).
```

Reply:
Thanks @nan-wang, Done. Could you approve and merge perhaps?